lang/id/index.md

```diff
@@ -0,0 +1,340 @@
+---
+title: Semantic Versioning 2.0.0
+language: id
+---
+
+Semantic Versioning 2.0.0
+=============================
+
+Ringkasan
+-------
+
+Diberi nomor versi MAJOR.MINOR.PATCH, kenaikan pada:
+
+1. versi MAJOR ketika anda membuat perubahan kompatibilitas API,
+1. versi MINOR ketika anda menambah fungsionalitas pada gaya kompatibilitas sebelumnya dan,
+1. versi PATCH ketika anda membuat perbaikan bug untuk kompatibilitas sebelumnya.
+
+label tambahan untuk pre-release dan build metadata tersedia sebagai ekstensi
+untuk format MAJOR.MINOR.PATCH.
+
+Pengantar
+------------
+
+Dalam dunia manajemen software terdapat tempat menakutkan yang disebut
+"dependency hell" Semakin besar sistem anda tumbuh dan semakin paket anda
+diintegrasikan ke dalam perangkat lunak anda, semakin besar kemungkinan anda
+untuk menemukan diri anda suatu hari pada lingkaran keputusasaan ini.
+
+Pada sistem dengan banyak dependensi, merilis versi paket baru dapat dengan cepat
+menjadi mimpi buruk. Jika spesifikasi dependensi terlalu ketat, anda akan masuk ke
+bahaya dari "version lock" (ketidakmampuan untuk meng-upgrade paket tanpa harus
+merilis versi baru dari setiap paket terkait). Jika dependensi
+ditentukan terlalu longgar, anda pasti akan digigit oleh "version promiscuity"
+(dengan asumsi kompatibilitas dengan versi yang lebih di masa mendatang lebih dari wajar).
+Dependency hell tempat anda berada ketika "version lock" dan/atau "version promiscuity"
+mencegah anda dari kemudahan dan keamanan pergerakan projek anda kedepan.
+
+Sebagai solusi untuk masalah ini, saya mengusulkan seperangkat aturan dan
+persyaratan yang mendikte bagaimana nomor versi ditugaskan dan bertambah.
+Aturan-aturan ini didasarkan pada metode yang ada dan banyak digunakan,
+tetapi tidak harus terbatas pada pre-existing praktek umum luas yang digunakan
+pada kedua perangkat lunak closed dan open-source. Untuk sistem ini bekerja,
+anda harus terlebih dahulu mendeklarasikan API publik. Hal ini mungkin
+terdiri dari dokumentasi atau ditunjukan oleh kode itu sendiri. Bagaimanapun, itu adalah
+penting bahwa API ini menjadi jelas dan tepat. Setelah anda mengidentifikasi publik API
+anda, anda mengkomunikasikan perubahan dengan penambahan khusus untuk nomor versi anda.
+Pertimbangkan format versi X.Y.Z (Major.Minor.Patch). Perbaikan bug API tidak mempengaruhi
+kenaikan versi patch, penambahan/perubahan API kompatibel kenaikan versi minor, dan perubahan
+API sebelumnya kompatibel kenaikan versi major.
+
+Saya menyebut sistem ini "Semantic Versioning." berdasarkan skema ini, penomoran versi
+dan cara mereka mengubah harus menyampaikan makna tentang kode yang mendasari dan apa yang
+telah dimodifikasi dari satu versi ke yang berikutnya.
+
+
+Spesifikasi Semantic Versioning (SemVer)
+------------------------------------------
+
+Kata kunci "HARUS", "TIDAK HARUS", "DIPERLUKAN", "AKAN", "TIDAK AKAN", "SEHARUSNYA",
```

Comment:
"TIDAK HARUS" should be changed to TIDAK BOLEH so its not miss-leading, "TIDAK AKAN" may be changed to "TIDAK BISA"